packages/eslint-plugin-template/tests/rules/click-events-have-key-events/cases.ts

```diff
@@ -6,7 +6,11 @@ const messageId: MessageIds = 'clickEventsHaveKeyEvents';
 export const valid = [
   {
     // It should work when click events are associated with key events.
-    code: '<div (click)="onClick()" (keyup)="onKeyup()"></div>',
+    code: `
+        <div (click)="onClick()" (keyup)="onKeyup()"></div>
+        <div (keyup)="onKeyup()" (click)="onClick()"></div>
```

Comment:
Additionally found that the order of click and key events shows as invalid if the key event came before the click event. As a workaround, we're putting the key event after the click event, but based on the initial finding here with additional outputs, any event/output after the click event could show a false positive.